transmart-core-db/grails-app/services/org/transmartproject/db/clinical/MultidimensionalDataResourceService.groovy

```diff
@@ -89,9 +92,9 @@ class MultidimensionalDataResourceService extends AbstractDataResourceService im
      * @param constraints: (nullable) A list of Constraint-s. If null, selects all the data in the database.
      * @param dimensions: (nullable) A list of Dimension-s to select. Only dimensions valid for the selected studies
      * will actually be applied. If null, select all available dimensions.
-     *
-     * Not yet implemented:
-     * @param sort
+     * @param sort: (nullable) Either a list of dimensions, or a LinkedHashMap of dimensions to "asc" or "desc".
+     * Dimensions can be either dimension objects or their string names. Note: allowed sortings are limited if
+     * modifier dimensions are used.
```

Comment:
This description should be updated also in the core-api interface: [MultiDimensionalDataResource](https://github.com/thehyve/transmart-core/blob/sorting2/transmart-core-api/src/main/groovy/org/transmartproject/core/multidimquery/MultiDimensionalDataResource.groovy#L20)

Reply:
I'll update that